src/screens/Ticketing/Tickets/Ticket.tsx

```diff
@@ -67,8 +67,8 @@ const Ticket: React.FC<Props> = ({fareContract: fc, now}) => {
         <View style={styles.ticketInfoContainer}>
           <ThemeText style={styles.travellersText}>
             {fc.user_profiles.length > 1
-              ? `${fc.user_profiles.length} voksne`
-              : `1 voksen`}
+              ? `${fc.user_profiles.length} billetter`
```

Comment:
As of now we do not get the traveller types in the `FareContractTicket'. Talked with Torbjørn about this, and it will be changed to returning a `travellers` array which has reference to both the ticket type and traveller type.